doc/user/content/self-managed-deployments/installation/install-on-azure.md

```diff
@@ -0,0 +1,309 @@
+
+### Step 2: Configure Terraform Variables
+
+1. Create a `terraform.tfvars` file with the following variables:
+
+   - `subscription_id`: Azure subscription ID
+   - `resource_group_name`: Name for the resource group to create (e.g.
+     `mz-demo-rg`)
+   - `name_prefix`: Prefix for all resource names (e.g., `simple-demo`)
+   - `location`: Azure region for deployment (e.g., `westus2`)
+   - `license_key`: Materialize license key
+   - `tags`: Map of tags to apply to resources
+
+   ```hcl
+   subscription_id     = "your-subscription-id"
+   resource_group_name = "mz-demo-rg"
+   name_prefix         = "simple-demo"
+   location            = "westus2"
+   license_key         = "your-materialize-license-key"
+   tags = {
+     environment = "demo"
+   }
+   ```
+
+### Step 3: Apply the Terraform
+
+1. Initialize the Terraform directory to download the required providers
+   and modules:
+
+   ```bash
+   terraform init
+   ```
+
+1. Apply the Terraform configuration to create the infrastructure.
+
+   - To deploy with the default **internal NLB** for Materialize access:
+
+   ```bash
+   terraform apply
+   ```
+
+   - To deploy with <red>**public NLB**</red> for Materialize access:
+
+   ```bash
+   terraform apply -var="internal=false"
+   ```
+
+   If you are satisfied with the planned changes, type `yes` when prompted
+   to proceed.
+
+1. From the output, you will need the following field(s) to connect:
+   - `console_load_balancer_ip` for the Materialize Console
+   - `balancerd_load_balancer_ip` to connect PostgreSQL-compatible clients/drivers.
+
+   ```bash
+   terraform output -raw <field_name>
+   ```
+
+   {{< tip >}}
+   Your shell may show an ending marker (such as `%`) because the
+   output did not end with a newline. Do not include the marker when using the value.
+   {{< /tip >}}
+
+1. Configure `kubectl` to connect to your cluster using your:
+   - `resource_group_name`. Your
+     resource group name can be found in the Terraform output or in the
+     `terraform.tfvars` file.
+
+   - `akw_cluster_name`. Your cluster name can be found in the Terraform output.
+     For the sample example, your cluster name has the form `{prefix_name}-aks`;
+     e.g., simple-demo-aks`.
+
+   ```bash
+   # az aks get-credentials --resource-group <resource_group_name> --name <your-aks-cluster-name>
+   az aks get-credentials --resource-group $(terraform output -raw resource_group_name) --name $(terraform output -raw aks_cluster_name)
+   ```
+
+
+### Step 4. Optional. Verify the deployment.
+
+1. Check the status of your deployment:
+   {{% include-from-yaml data="self_managed/installation"
+   name="installation-verify-status" %}}
+
+### Step 5: Connect to Materialize
+
+{{< note >}}
+
+If using an **internal Network Load Balancer (NLB)** for your Materialize
+access, you can connect from inside the same VPC or from networks that are
+privately connected to it.
+
+{{< /note >}}
+
+#### Connect using the Materialize Console
```

Comment:
FYI -- the azure/google simple examples do not have auth enabled like aws's simple example.